.zenodo.json

```diff
@@ -761,6 +767,9 @@
             "name": "Freek Verstringe",
             "affiliation": "Royal Observatory of Belgium"
         },
+        {
+            "name": "FreyaJain"
```

Comment:
@FreyaJain 

Would you like your name, affiliation and orcid added to the release?